motif-docs/pages/guides/config/monorepos.mdx

```diff
@@ -27,15 +125,15 @@ If you want use a `nixpacks.toml` file in the packages of a monorepo, you'll nee
       envVariables": {
         "NIXPACKS_CONFIG_FILE": "./apps/frontend/nixpacks.toml"
       }
-      ...restOfConfig
+      ...
     },
     {
-      id: "backend",
+      id: "api",
       buildType: "nixpacks",
       envVariables": {
-        "NIXPACKS_CONFIG_FILE": "./apps/backend/nixpacks.toml"
+        "NIXPACKS_CONFIG_FILE": "./packages/api/nixpacks.toml"
       }
-      ...restOfConfig
+      ...
     },
   ],
 }
```

Comment:
Instead of using `NIXPACKS_CONFIG_FILE`, they should define `basePath` to the service, and then add `nixpacks.toml` in that basePath location.